testing/regress/hpcc/util/ecl/command.py

```diff
@@ -78,7 +78,9 @@ def runCmd(self, cmd, cluster, eclfile, report, **kwargs):
 
             args = args + eclfile.getStoredInputParameters()
 
-            args.append('-XTargetIP=' + server)
+            if '-XTargetIP' not in "".join(args):
+                logging.debug("%3d. results:'%s'", eclfile.getTaskId(),  "'-XTargetIP' is not in args")
+                args.append('-XTargetIP=' + server)
```

Comment:
As mentioned in the JIRA comments:
```
1) Change ecl-test so it does not set TargetIP, i.e. TargetIP becomes a regular parameter like any other.
The tests default to ".", which is correct for a vanilla setup.
```

It doesn't need setting, the test default to "." if not set.
So it should be set, nor passed unless it is passed explicitly or set ecl-test.json